apps/firecrawler/firecrawl_mcp/tools/crawl.py

```
@@ -168,47 +174,92 @@ async def _handle_crawl_start(
+                    f"Crawl started successfully for {target_url} with job ID: {crawl_response.id}",
+                )
+                successful_jobs.append((target_url, crawl_response))
+
+            except FirecrawlError as e:
+                error_context = f"tool=crawl, url={target_url}, limit={limit}"
+                mcp_error = handle_firecrawl_error(e, error_context)
+                message = str(mcp_error)
+                await ctx.error(f"Firecrawl API error during crawl: {message}")
+                failed_jobs.append({"url": target_url, "error": message})
+
+            except Exception as e:
```

Pylint (3.3.8)

[warning] Catching too general exception Exception

(W0718)